projects/packages/backup/src/js/components/Admin.js

```diff
@@ -65,10 +87,48 @@ const Admin = () => {
 		return capabilities.includes( 'backup' );
 	};
 
+	// Check if the site has a successful restore not older than 15 days
+	const hasRecentSuccesfulRestore = () => {
+		if ( restores[ 0 ] && restores[ 0 ].status === 'finished' ) {
+			// Number of days we consider the restore recent
+			const maxDays = 15;
+			const daysDifference = ( new Date() - Date.parse( restores[ 0 ].when ) ) / 86400000;
+
+			if ( daysDifference < maxDays ) {
+				return true;
+			}
+		}
+		return false;
+	};
+
+	const hasThreeMonthsSub = () => {
+		// TODO add logic for multiple purchases
+		// check if older purchases come first or last on sites with multiple purchases
+		for ( const purchase of currentPurchases ) {
+			const isBackupPlan =
+				JETPACK_BACKUP_PRODUCTS.includes( purchase.product_slug ) ||
+				JETPACK_PLANS_WITH_BACKUP.includes( purchase.product_slug );
+			const isActive = purchase.subscription_status === 'active';
+
+			if ( isBackupPlan && isActive ) {
+				const subscriptionTimeInDays =
+					( new Date() - Date.parse( purchase.subscribed_date ) ) / 86400000;
+				if ( subscriptionTimeInDays > 90 ) {
+					return true;
+				}
+			}
+		}
+		return false;
+	};
+
 	const sendToCart = () => {
 		window.location.href = getRedirectUrl( 'backup-plugin-upgrade-10gb', { site: domain } );
 	};
 
+	const sendToReview = () => {
```

Comment:
These callback functions (sendToReview and sendToCart) should be defined using the [useCallback](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-element/#:~:text=https%3A//reactjs.org/docs/hooks%2Dreference.html%23usecallback) hook. It can be imported from `@wordpress/element`.